src/app/contextBroker/contextBroker.cpp

```diff
@@ -64,6 +63,7 @@
 #include <string>
 #include <vector>
 #include <limits.h>
+#include <stdio.h>             /* snprintf                                    */
```

Comment:
Why not put it back where it was (11 lines before), and use C++ comments, indented like for `unistd.h` and `fcntl.h` ?
Just for "prettiness" :)

Reply:
Fixed in 7ea8ab6